server/index.js

```diff
@@ -14,6 +14,4 @@ app.use('/', router);
 
 app.set('port', process.env.PORT || 7878);
```

Comment:
interesting choice to use app.set. are you just trying to avoid creating a local variable to hold the port number?

Reply:
This was actually used in one of our server sprints. I think chatterbox server. I just adopted it since haha